### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by akshaysonvane**
Bugfix for Issue #364 https://github.com/marklogic/marklogic-sesame/issues/364
Added "Unsupported RDF Format."  MarkLogicRdf4jException for RDF formats that are not supported by MarkLogic.

### CODE REVIEW

File: marklogic-rdf4j/gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 group=com.marklogic
-version=2.0.0
+version=1.0.0
```
Comment (grechaw) [inline marklogic-rdf4j/gradle.properties]: good catch

File: marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java [hunk 2]
```diff
@@ -154,7 +148,7 @@ public MarkLogicRepository(String host, int port, String user, String password,
     /**
      * constructor inited with java api client DatabaseClient
      *
-     * @param databaseClient
+     * @param databaseClient the java api client to be used
```
Comment (grechaw) [inline marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java]: Here's the corresponding javadoc from jena:

```
/**
     * Constructor.
     * 
     * @param client
     *            a Java Client API DatabaseClient. Can be made with
     *            com.marklogic.client.DatabaseClientFactory
     */
```

If you include packages it's easier for somebody to learn what's going on with this library

File: marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java [hunk 3]
```diff
@@ -286,7 +274,7 @@ public boolean isWritable() throws RepositoryException {
     }
 
     /**
-     * returns a MarkLogicConnection object which is the entry point to
+     * Returns a MarkLogicConnection object which is the entry point to
```
Comment (grechaw) [inline marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java]: Use @return annotation for returns.

File: marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java [hunk 4]
```diff
@@ -322,17 +310,17 @@ public synchronized MarkLogicClient getMarkLogicClient() {
     }
 
     /**
-     * sets MarkLogicClient used by this repository
+     * Sets MarkLogicClient used by this repository.
      *
-     * @param client
+     * @param client the MarkLogicClient to be used
```
Comment (grechaw) [inline marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java]: I know it's hard to say something useful sometimes, esp. for a setter, but if the sentence is meaningless probably you shouldn't bother.  I'd say here it's important to know that the MarkLogicClient mediates all interaction with the database.

**Comment by akshaysonvane** [comment]
@grechaw, PR merged without any issues. I will update the javadocs as per your comments in my local and make a new PR, once done.

**Pull request merged**